lib/emulators/avd.js

```diff
@@ -227,13 +228,17 @@ exports.start = function start(config, emu, opts, callback) {
 					'-port', port,                                   // TCP port that will be used for the console
 				];
 
-				if (opts.partitionSize !== undefined) {
-					args.push('-partition-size', opts.partitionSize);  // system/data partition size in MBs
-				}
+				const cpuCore = os.cpus();
+				const isM1 = cpuCore[0].model.includes('Apple');
+				if (!isM1) {
+					if (opts.partitionSize !== undefined) {
```

Comment:
I don't think the `partitionSize` needs to be in this conditional. It doesn't seem to affect anything. Is there a reason it's in this block? Something specific I should test?

Reply:
It was there before and it worked for non Arm Macs as it was setting one of those values in the emulator (I think SD card) if you don't let Studio manage it:
![Screenshot_20240408_085153](https://github.com/tidev/node-titanium-sdk/assets/4334997/f744ed69-55f5-4118-9b85-08cb9b3a9409)

but if that is set the Arm emulator would start with a black screen if you try to set it. That's why I moved it into a `isM1` check. I didn't want to change anything for other CPU as that seem to work like it is.

Personally I never used the parameter as I create emulators with studio and set all the sizes in there.

Reply:
I think this is looking good! Appreciate the iteration. Let's ship this and see what happens. 🎉